Chummer/Backend/Debugging/CrashHandler.cs

```diff
@@ -192,16 +192,20 @@ public static void WebMiniDumpHandler(Exception ex)
                 byte[] info = new UTF8Encoding(true).GetBytes(dump.SerializeBase64());
                 File.WriteAllBytes(Path.Combine(Utils.GetStartupPath, "json.txt"), info);
 
-                if (GlobalOptions.UseLoggingApplicationInsights)
+                if (GlobalOptions.UseLoggingApplicationInsights == EnumUseLoggingApplicationInsights.yes
+                    || GlobalOptions.UseLoggingApplicationInsights == EnumUseLoggingApplicationInsights.crashes)
                 {
                     if (Program.TelemetryClient != null)
                     {
+                        var oldvalue = GlobalOptions.UseLoggingApplicationInsights;
+                        GlobalOptions.UseLoggingApplicationInsights = EnumUseLoggingApplicationInsights.yes;
```

Comment:
This is _not_ the correct way of enabling telemetry for crashes. 
Properties are supposed to be fancy variables, this is specifically relying on side effects, of writing to a property, without even a comment. 

I would simply write to the telemetry variable, if this is not a good solution for some reason, create a TelemetrySomething that both Options and this can interact with.

Reply:
That actually was already legacy. I got rid of it.